# Solicitud
uint64 timestamp
bool store
---
# Respuesta
bool success
